R/datatypes-s3-base.R

```diff
@@ -38,7 +38,7 @@
 
 #' @export
 print.polars_dtype <- function(x, ...) {
-  x$`_dt`$as_str() |>
+  x$as_str() |>
```

Comment:
Necessary now that `as_str()` is properly added in datatypes functions on the R side. (The alternative would be to manually add `abbreviated = FALSE`, which isn't better)